mlrun/runtimes/function.py

```diff
@@ -859,6 +860,7 @@ def invoke(
         :param force_external_address:   use the external ingress URL
         :param auth_info: service AuthInfo
         :param mock:     use mock server vs a real Nuclio function (for local simulations)
+        :param kwargs:   allow the user to pass any parameter supported in requests.request method
```

Comment:
@GiladShapira94 - let's add a link to the `requests.request` API docs here (https://requests.readthedocs.io/en/latest/api/#requests.request)

Reply:
Fix 👍